The user guide is installed in /usr/share/backup-manager-doc.
This is the recommended canonical Debian location.  See
http://bugs.debian.org/433011, http://bugs.debian.org/528161.

Provided by: Sven Joachim <svenjoac@gmx.de>

---
 doc/Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/doc/Makefile
+++ b/doc/Makefile
@@ -1,7 +1,7 @@
 
 BASE	= user-guide
 SOURCE	= $(BASE).sgml
-DEST    = $(DESTDIR)/usr/share/backup-manager/doc
+DEST    = $(DESTDIR)/usr/share/backup-manager-doc
 
 all: pdf txt html
 install: all
